custom_components/zaptec/__init__.py

```diff
@@ -71,24 +71,13 @@ async def async_setup(hass: HomeAssistantType, config: ConfigType) -> bool:
 async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry) -> bool:
     """Set up zaptec as config entry."""
     await _dry_setup(hass, entry.data)
-
-    for platform in PLATFORMS:
-        hass.async_create_task(
```

Comment:
I'm not sure this is a proper fix, im memory serves me there was some issue with import if we didn't used async_create_task.

Reply:
The only difference between the old implementation and the new, is that the latter runs `asyncio.gather()` on the execution of the tasks. The old code simply creates the tasks and leaves them to their own devices. A hang here will hang the setup.

(The new is AFAICS the preferred way to setup the platforms)

Reply:
Sweet!